opennlp-tools/src/main/java/opennlp/tools/ml/model/AbstractDataIndexer.java

```diff
@@ -33,6 +35,18 @@
  */
 public abstract class AbstractDataIndexer implements DataIndexer {
 
+  public static final String CUTOFF_PARAM = AbstractTrainer.CUTOFF_PARAM;
+  public static final int CUTOFF_DEFAULT = AbstractTrainer.CUTOFF_DEFAULT;
+  
+  public static final String SORT_PARAM = "sort";
+  public static final boolean SORT_DEFAULT = true;
+
+  PluggableParameters parameters;
+
+  public void init(Map<String,String> indexingParameters,Map<String, String> reportMap){
```

Comment:
leave a space between the comma and next parameter, change to 

`indexingParameters, Map<String, String> `